<launch>
  <!--include file="$(find epos_ros)/config/ft_config.yaml"/-->


  <node pkg="epos_ros" type="test_node" name="epos_motor_command" respawn="true" output="screen"/>
  <node pkg="epos_ros" type="ft_test_control_node" name="ft_test_control_node" respawn="true"/>
  <!--node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard" name="twist_keyboard" respawn="true" /-->



</launch>
